cmd/thor/utils.go

```diff
@@ -395,6 +395,20 @@ func masterKeyPath(ctx *cli.Context) (string, error) {
 }
 
 func loadNodeMaster(ctx *cli.Context) (*node.Master, error) {
+
+	masterKey := ctx.String(masterKeyFlag.Name)
+	if masterKey != "" {
```

Comment:
Looks good! A small nit to avoid code repetition sugestion would be a small refactor. I think perhaps something like:
1. key is set by flag
2. otherwise key is set by path
3 if key is set then test beneficiary and return masterkey

Reply:
@otherview I made a commit to refactor this